raiden_contracts/tests/test_channel_unlock.py

```diff
@@ -541,7 +544,7 @@ def test_channel_unlock_no_locked_amount_fail(
 ) -> None:
     """After settleChannel() is called with zero locked amount, unlock() calls fail"""
     (A, B) = get_accounts(2)
-    settle_timeout = 8
+    settle_timeout = TEST_SETTLE_TIMEOUT
```

Comment:
What is the purpose of this variable assignment? Doesn't seem to add any value to the code and is rather annoying. I think I even removed it at some places when I was editing the tests to fix them.